### ISSUE TITLE
ansible-lint check is failing

### ISSUE THREAD

**Issue by sxa** [adoptium/infrastructure#2426]
Appears to be on multiple PRs in this repository, so unlikely to be specific to any particular one and therefore needs to be resolved independently of any PR:

```
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.1/x64/bin/ansible-lint", line 8, in <module>
    sys.exit(_run_cli_entrypoint())
  File "/opt/hostedtoolcache/Python/3.10.1/x64/lib/python3.10/site-packages/ansiblelint/__main__.py", line 310, in _run_cli_entrypoint
    sys.exit(main(sys.argv))
  File "/opt/hostedtoolcache/Python/3.10.1/x64/lib/python3.10/site-packages/ansiblelint/__main__.py", line 214, in main
    from ansiblelint.generate_docs import rules_as_rich, rules_as_rst, rules_as_str
  File "/opt/hostedtoolcache/Python/3.10.1/x64/lib/python3.10/site-packages/ansiblelint/generate_docs.py", line 6, in <module>
    from rich.console import render_group
ImportError: cannot import name 'render_group' from 'rich.console' (/opt/hostedtoolcache/Python/3.10.1/x64/lib/python3.10/site-packages/rich/console.py)
Error: Process completed with exit code 1.
```

- Link to any log file showing the problem:

- Please describe the issue:

**Comment by Willsparker** [comment]
For some reason, the Linter has decided to fix itself - in both #2420 and #2427 , the check has passed.

**Comment by Willsparker** [comment]
Did a quick re-test on my PR (#2424 ) and it passed as expected. Closing
